protos/orchestrator_service.proto

```diff
@@ -266,6 +266,18 @@ message CreateInstanceRequest {
     google.protobuf.StringValue version = 3;
     google.protobuf.StringValue input = 4;
     google.protobuf.Timestamp scheduledStartTimestamp = 5;
+    CreateInstanceOption createInstanceOption = 6;
+}
+
+message CreateInstanceOption {
+    repeated OrchestrationStatus operationStatus = 1;
+    CreateOrchestrationAction action = 2;
+}
+
+enum CreateOrchestrationAction {
+    THROW = 0;
```

Comment:
Throw is a language-specific concept. I wonder if we should change this to `ERROR` or something else that's more language agnostic.